config.default.jsonc

```diff
@@ -1,26 +1,113 @@
+    "slack": [
+      "https://hooks.slack.com/services/T00000000/B00000000/XXXXXXXXXXXXXXXXXXXXXXXX"
+    ],
+
+    // Array of Discord webhook URLs for sending notifications (optional)
+    "discord": [
+      "https://discord.com/api/webhooks/123456789012345678/aBCdeFg9h0iJKl1-_mNoPqRST2uvwXYZ3ab4cDefgH5ijklmnOPQrsTuvWxYZaBC-de_"
+    ],
+
+    // Secret passphrase for ADAMANT notifications (optional)
+    "adamantPassphrase": "apple banana...",
+    // Array of ADAMANT addresses for sending notifications (optional)
+    "adamant": ["U12345678901234567890"]
+  },
+  "log_level": "log", // Specifies the verbosity of logs (none, log, warn, error)
+
+  "moex": {
+    // Enable or disable Moex API (optional)
+    "enabled": true,
+    // API URL
+    "url": "https://rusdoor.adamant.im/securities.jsonp",
+    // Record of fiat pairs and their codes to fetch rates from MOEX
+    "codes": {
+      "USD/RUB": "USDRUB_TOM",
+      "EUR/RUB": "EURRUB_TOM",
+      "CNY/RUB": "CNYRUB_TOM"
+      // ...
+    }
   },
-  "log_level": "log",
+
+  "base_coins": ["USD", "RUB", "EUR", "CNY", "JPY", "BTC", "ETH"], // List of base coins for calculating all available pairs
+
+  "currency_api": {
+    // Enable or disable CurrencyApi (optional)
+    "enabled": true,
+    // API URL
+    "url": "https://cdn.jsdelivr.net/npm/@fawazahmed0/currency-api@latest/v1/currencies/usd.json",
+    // List of the coins to exclude from the base_coins for CurrencyApi (optional)
+    "skip": ["USD", "ETH"]
+  },
+
+  "exchange_rate_host": {
+    "enabled": false, // Enable or disable ExchangeRate API (optional)
+    "api_key": "API key for ExchangeRate.",
+    "skip": ["USD", "ETH"] // List of the coins to exclude from the base_coins for the ExchangeRateHost (optional)
+  },
+
   "coinmarketcap": {
-    "api_key": "hello_world",
-    "coins": ["BTC", "ETH"],
+    "enabled": false, // Enable or disable CoinMarketCap API (optional)
+    "api_key": "API key for CoinMarketCap",
+    "coins": [
+      // List of coins to fetch rates from CoinMarketCap (optional, deprecated)
+      "BTC",
+      "ETH",
+      "ADM"
+      // ...
+    ],
     "ids": {
-      "BTC": 102,
-    },
+      // Record of CoinMarketCap SymbolId/UCID pairs for specific coin rates (optional, preferred)
+      "BTC": 1,
+      "ETH": 1027,
+      "ADM": 3703
+      // ...
+    }
   },
-  "base_coins": ["USD", "RUB", "EUR", "CNY", "JPY", "BTC", "ETH", "KRW"],
-  "moex": {
-    "USD/RUB": "USDRUB_TOM",
-    "EUR/RUB": "EURRUB_TOM",
-    "GBR/RUB": "GBPRUB_TOM",
-    "CNY/RUB": "CNYRUB_TOM",
-    "JPY/RUB": "JPYRUB_TOM",
+
+  "cryptocompare": {
+    "enabled": true, // Enable or disable CryptoCompare API (optional)
+    "api_key": "API key for CryptoCompare",
+    "coins": [
+      // List of coins to fetch rates from CryptoCompare (optional)
+      "BTC",
+      "ETH",
+      "ADM"
+      // ...
+    ]
   },
+
+  "coingecko": {
+    "enabled": true, // Enable or disable CoinGecko API (optional)
```

Comment:
Isn't Coingecko requires an API key? If yes, add it to the config. Also disable it by default.